template-parts/wholesaler/content-contact-person.php

```diff
@@ -9,17 +9,17 @@
 
     <?php if ( get_field( "contact_photo" ) ): ?>
     <img
-      class="align-self-start rounded-top-left rounded-bottom-left mr-2"
+      class="align-self-start rounded-top-left rounded-bottom-left"
       src="<?php echo get_field( "contact_photo" )[ "sizes" ][ "thumbnail" ]; ?>"
       <?php if ( get_field( "contact_full_name" ) ): ?>
         alt="<?php the_field( "contact_full_name" ); ?>"
       <?php endif; ?>
-      height="100"
-      width="100"
+      height="104"
```

Comment:
U mě v chromu je 104 na půlpixel, takže to dělá mezi obrázkem a boxem mezeru. Jen FYI, krom přidání margin-bottom:-1px to asi vyřešit nejde.